src/Config.example

```diff
@@ -91,68 +106,227 @@ maximum array size : 15000
 maximum buffer size : 400000
 
 # Max size for a mapping
-maximum mapping size : 15000
-
-# Max inherit chain size
-inherit chain size : 30
+maximum mapping size : 150000
 
 # maximum length of a string variable
 maximum string length : 200000
 
-# Max size of a file allowed to be read by 'read_file()'.
-maximum read file size : 200000
+# Maximum number of bits in a bit field. They are stored in printable
+# strings, 6 bits per byte.
+maximum bits in a bitfield : 12000
 
 # max number of bytes you allow to be read and written with read_bytes 
 # and write_bytes
 maximum byte transfer : 200000
 
-# Reserve an extra memory area from malloc(), to free when we run out
-# of memory and allow the mudlib to shutdown.
-# If this value is 0, no area will be reserved.
-reserved size : 0
-
-# Define the size of the shared string hash table.  This number should
-# a prime, probably between 1000 and 30000; if you set it to about 1/5
-# of the number of distinct strings you have, you will get a hit ratio
-# (number of comparisons to find a string) very close to 1, as found strings
-# are automatically moved to the head of a hash chain.  You will never
-# need more, and you will still get good results with a smaller table.
+# Max size of a file allowed to be read by 'read_file()'.
+maximum read file size : 200000
+
+# Set the size of the shared string hash table.  This number should a prime,
+# probably between 1000 and 30000; if you set it to about 1/5 of the number
+# of distinct strings you have, you will get a hit ratio (number of
+# comparisons to find a string) very close to 1, as found strings are
+# automatically moved to the head of a hash chain.  You will never need more,
+# and you will still get good results with a smaller table.
 hash table size : 7001
 
-# Object hash table size.
-# Define this like you did with the strings; probably set to about 1/4 of
-# the number of objects in a game, as the distribution of accesses to
-# objects is somewhat more uniform than that of strings.
+# Set this like you did with the strings; probably set to about 1/4 of the
+# number of objects in a game, as the distribution of accesses to objects is
+# somewhat more uniform than that of strings.
 object table size : 1501
 
-# default no-matching-action message
-default fail message : What?
-
-# default message when error() occurs (optional)
-default error message :
-
-# the external ports we support
-external_port_1 : telnet 4000
-external_port_2 : binary 4001
-
-###############################################################################
-#          The following aren't currently used or implemented (yet)           #
-###############################################################################
+# There is a hash table for living objects, used by find_living().
+# This must be one of 4, 16, 64, 256, 1024, 4096
+living hash table size : 256
+
+#
+gametick msec : 1000
+
+# Set heartbeat interval in microseconds (us).
```

Comment:
the docs here is obviously wrong. This config entry has changed unit type to "msec" . please delete all these obsolete lines..